ui/components/ui/tabs/tabs.types.ts

```diff
@@ -47,14 +41,14 @@ export type TabsProps<TKey extends string = string> = Omit<
   onTabClick?: (tabKey: TKey) => void;
   /** Tab components to render */
   children: React.ReactNode;
-  /** CSS class name for the tabs container */
-  tabsClassName?: string;
```

Comment:
Not used

<img width="315" height="249" alt="Screenshot 2025-09-18 at 2 17 31 PM" src="https://github.com/user-attachments/assets/4e9dc601-4bd5-4011-94a7-ba68fd2b1794" />